pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -97,9 +97,18 @@ func (o *capability) RegisterToWorkflow(ctx context.Context, request capabilitie
 		return err
 	}
 
+	if c.AggregationConfig == nil {
```

Comment:
We should default to `data_feeds_2_0` and `EVM` encoder. This will keep the demo Workflow specs smaller and when we need something different (Solana) we'll be able to showcase this much better.